streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestUtil.java

```diff
@@ -34,10 +34,34 @@ public class SmokeTestUtil {
 
     final static int END = Integer.MAX_VALUE;
 
+    static ProcessorSupplier<Object, Object, Void, Void> printTaskProcessorSupplier(final String topic) {
+        return printTaskProcessorSupplier(topic, "");
+    }
+
     static ProcessorSupplier<Object, Object, Void, Void> printProcessorSupplier(final String topic) {
         return printProcessorSupplier(topic, "");
     }
 
+    static ProcessorSupplier<Object, Object, Void, Void> printTaskProcessorSupplier(final String topic, final String name) {
```

Comment:
nit: this new processor is the same as the existing one except that it doesn't track or print the number of records processed, right? Would it be better to have a boolean to toggle the print behavior, rather than duplicating the rest of the processor code? (Not a big deal either way since it's not much code, but as a reader I had to spent some time determining/verifying that the print behavior is the only difference.)

Reply:
Yeah, it was a quick-and-dirty thing -- guess it might make sense to actually have a single Processor and let the original one inherit from the new one.